chainer/links/model/classifier.py

```diff
@@ -29,6 +29,23 @@ class Classifier(link.Chain):
         compute_accuracy (bool): If ``True``, compute accuracy on the forward
             computation. The default value is ``True``.
 
+    .. note::
+        This link uses :func:`chainer.softmax_cross_entropy` with
+        default arguments as a loss function (specified by ``lossfun``),
+        if users do not explicit change it. In particular, the loss function
+        does not support double backpropagation.
+        If you need second or higher order differentiation, you need to turn
+        it on with ``enable_double_backprop=True``:
+
+    .. admonition:: Example
```

Comment:
Can this example section be included in the above note? (It looks it is currently a separate section)

Reply:
Fixed